server/elasticsearch/src/main/resources/hawkular-apm-mapping.json

```diff
@@ -17,11 +17,6 @@
             }],
             "date_detection" : false,
             "numeric_detection": false,
-            "_timestamp": {
```

Comment:
It should be ms https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-timestamp-field.html and we set it to microseconds, so probably don't use it.